storagemarket/types/deal_state.go

```diff
@@ -58,6 +58,9 @@ type ProviderDealState struct {
 
 	// NBytesReceived is the number of bytes Received for this deal
 	NBytesReceived int64
+
+	// Keep unsealed copy of the data
+	FastRetrieval bool
```

Comment:
We need to add some code to make sure the FastRetrieval flag gets written to / read from the database when the ProviderDealState is written to the DB

Reply:
I will add that part